src/Uno.UI/UI/Xaml/Controls/CalendarView/CalendarViewBaseItemChrome.IBorderInfoProvider.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.UI.Xaml.Media;
+using Uno.UI.Xaml.Controls;
+
+namespace Microsoft.UI.Xaml.Controls;
+
+partial class CalendarViewBaseItem : IBorderInfoProvider
+{
+	Brush IBorderInfoProvider.Background
```

Comment:
This code appears to be duplicated in `UpdateChrome`. Consider letting `UpdateChrome` do `((IBorderInfoProvider)this).Background` (or extract this code to a method that both will call)